pkg/openshift/client.go

```diff
@@ -0,0 +1,120 @@
+package openshift
```

Comment:
Any reason for keeping this separate from cmd/ike/session/session_client?

Reply:
We can put this together. My thinking was that this one is a generic-purpose client which has no typed interfaces so `openshift` seemed to be a more natural place.

Reply:
Was not thinking about the location, but rather function. Minimally we could use the 'non typed' 'client' internally within the 'SessionClient' interface. It's our Resource so we shouldn't really be surprised by changes to it. hehe

Reply:
Ah ok, got it. Let me go over the code and see - there's definitely some dupl in a way the client is set up for example. We can have both typed and generic in one place, why not.

Reply:
I opened #150 issue to clean up the client stuff.